fpsim/locations/senegal.py

```diff
@@ -38,6 +38,9 @@ def scalar_pars():
         'breastfeeding_dur_mu'   : 19.66828, # Location parameter of gumbel distribution. Requires children's recode DHS file, see data_processing/breastfeedin_stats.R 
         'breastfeeding_dur_beta' : 7.2585, # Scale parameter of gumbel distribution. Requires children's recode DHS file, see data_processing/breastfeedin_stats.R 
         'max_lam_dur'            : 5, # Duration of lactational amenorrhea
+        'short_int'              : 24,# Duration of a short birth interval between live births in months
+        'age_low'                : 0, # age limit for tracking the age-specific short birth interval 
+        'age_high'               : 20,# age limit for tracking the age-specific short birth interval
 
```

Comment:
Is this for tracking, or for plotting only? I would like a more specific par name here so that when others are doing their own calibrations, it is not so easily confused with things like method age or other age related pars.

Reply:
They are for both plotting and tracking. 
How about age_low_short_int and  age_high_short_int?

Reply:
How about low_age_short_int and high_age_short_int? For some reason that seems more legible to me.

Reply:
Sounds good. I'll make the changes shortly.